src/providers/ProfileModalContext.js

```diff
@@ -0,0 +1,3 @@
+import React from 'react';
+
+export const ProfileModalContext = React.createContext();
```

Comment:
Awesome that we can cover `provider` separating this. However, wouldn't this be better placed in somewhere else like `src/contexts` and ignore path from `testCoverage` because it will never be tested.

Reply:
yep. I did what you've suggested. added `/src/contexts/` in jest's `coveragePathIgnorePatterns` option.